Out of proc code coverage Take 2

ThomasSimeroth opened the pull request:
I got tired of waiting for #89 to land, so I took matters into my own hands. All I did was change some of the test scripts. All credit for the actual code goes to @ProTip .

Summary of changes:

- Add testURL to Jest config to fix Jest bug. See jsdom/jsdom#2304 for more details.
- Add test in diff-snapshot.spec.js for size mismatch
- Change customSnapshotDir to rely on path for custom default test
- Address unaddressed developer feedback on #89

I'm excited to see this get merged in. Let me know if you have any comments or concerns.

Edit: I had to redo my commits, in order to make them appear on my Github account. Now that that's done, the licensing agreement will actually pass.

anescobar1991 commented on package.json:
```diff
@@ -33,7 +33,8 @@
       "<rootDir>/examples"
-    ]
+    ],
+    "testURL": "http://localhost/"
```
Instead of adding this can you update `amex-jest-preset` to 5.0.0? It no longer uses `jsdom` as its test environment which is fine since we don't need it in this package and will solve the `localStorage` bug with Jest.

anescobar1991 commented on __tests__/diff-snapshot.spec.js:
```diff
@@ -15,26 +15,62 @@
     const mockWriteFileSync = jest.fn();
     const mockPixelMatch = jest.fn();
 
+
```
Remove this extra line please.

ThomasSimeroth commented:
Hmm... Not sure why the travis build stalled when I implemented the changes. I thought it might have been testURL, but apparently not. I think it was a problem with travis itself, but I'm not sure. Either way, I've implemented the changes you asked for.

ThomasSimeroth commented:
@10xLaCroixDrinker Any chance for a review? I'd like to get this merged soon.
